docker/Dockerfile

```diff
@@ -6,18 +6,16 @@ ENV ORION_USER orion
 ENV GIT_REV_ORION develop
 ENV CLEAN_DEV_TOOLS 1
 
-RUN adduser --comment "${ORION_USER}" ${ORION_USER}
-
 WORKDIR /opt
 
 RUN \
+    adduser --comment "${ORION_USER}" ${ORION_USER} && \
     # Install dependencies
     yum -y install epel-release && \
     yum -y install \
       boost-devel \
       bzip2 \
       cmake \
-      epel-release \
```

Comment:
I though that some epel-release dependencies were required, but I understand that if you have tested with this removed everthing works ok with `FROM centos:centos6.6`.